debian/control

```diff
@@ -3,14 +3,14 @@ Section: non-free/net
 Priority: extra
 Maintainer: Hannu Varjoranta <hannu.varjoranta@spotify.com>
 Build-Depends:
-  python (>= 2.7),
-  libpq-dev (<<9.7),
+  python3 (>= 3.5),
+  libpq-dev,
   debhelper (>= 9),
   dh-virtualenv (>= 1.0),
```

Comment:
I think we need to include `dh-systemd (>= 1.5) ` here as well, given the repo has a compat <10